packages/features/settings/outOfOffice/CreateOrEditOutOfOfficeModal.tsx

```diff
@@ -104,8 +104,8 @@ export const CreateOrEditOutOfOfficeEntryModal = ({
       ? currentlyEditingOutOfOfficeEntry
       : {
           dateRange: {
-            startDate: dayjs().utc().startOf("d").toDate(),
-            endDate: dayjs().utc().add(1, "d").endOf("d").toDate(),
+            startDate: dayjs().startOf("d").toDate(),
+            endDate: dayjs().startOf("d").add(2, "d").toDate(),
```

Comment:
`endDate` is just a date, and not a time. So we don't need to use `endOf` here. I put 2 days between `startDate` and `endDate` (the same as the current behavior)

Also, we should keep using the local Date instance on the browser side, and it will be converted on the server side. So no need for utc() method here, and it's rather wrong.